pkg/apps/database/postgres/postgres.go

```diff
@@ -109,7 +109,19 @@ func (p *Postgres) GetPostgresReplicationController() (*components.ReplicationCo
 		initContainers = append(initContainers, postgresInitContainerConfig)
 	}
 
-	pod, err := util.CreatePod(postgresName, "", postgresVolumes, []*util.Container{postgresExternalContainerConfig}, initContainers, []horizonapi.PodAffinityConfig{}, p.Labels)
+	pod, err := util.CreatePod(postgresName,
+		&util.PodConfig{
+			ServiceAccount:         "",
+			Volumes:                postgresVolumes,
+			Containers:             []*util.Container{postgresExternalContainerConfig},
+			InitContainers:         initContainers,
+			PodAffinityConfigs:     []*util.PodAffinityConfig{},
+			PodAntiAffinityConfigs: []*util.PodAffinityConfig{},
+			NodeAffinityConfigs:    []*util.NodeAffinityConfig{},
+			ImagePullSecrets:       []string{},
```

Comment:
No need to provide this field if it's empty

Reply:
done